rc/base/lisp.kak

```diff
@@ -26,17 +26,31 @@ add-highlighter shared/lisp/code/ regex (\b\d+)?\.\d+([eEsSfFdDlL]\d+)?\b 0:valu
 # Commands
 # ‾‾‾‾‾‾‾‾
 
-define-command -hidden lisp-filter-around-selections %{
+define-command -override -hidden lisp-filter-around-selections %{
```

Comment:
I think we'd rather rename this function for use in the `lisp.kak` module, rather than try to override a command from another module.

Reply:
Oops.  This wasn't actually overridden, I just had this here for development.  Removed.